### PR TITLE
Update picker

### PR DESCRIPTION
**Pull request by Asharma34**
update picker.js

### CODE REVIEW

**Comment by Aniket21mathur** [comment]
Hii @Asharma34! Thanks for contributing.

Please see te suggested changes :)

File: assets/js/picker.js [hunk 1]
```diff
@@ -18,4 +18,4 @@ window.onload = function() {
 	$hidePicker.onclick = function () {
```
Comment (Aniket21mathur) [inline assets/js/picker.js]: I think this need to be renamed as well.

**Review by Aniket21mathur — CHANGES_REQUESTED** [review]

**Comment by Aniket21mathur** [comment]
Looks good 👍

**Comment by Asharma34** [comment]
Thanks @Aniket21mathur

**Pull request merged**